website/docs/reference/warehouse-setups/infer-setup.md

```diff
@@ -0,0 +1,125 @@
+    <li><strong>PyPI package</strong>: <code>{frontMatter.meta.pypi_package}</code> <a href={`https://badge.fury.io/py/${frontMatter.meta.pypi_package}`}><img src={`https://badge.fury.io/py/${frontMatter.meta.pypi_package}.svg`}/></a></li>
+    <li><strong>Slack channel</strong>: <a href={frontMatter.meta.slack_channel_link}>{frontMatter.meta.slack_channel_name}</a></li>
+    <li><strong>Supported dbt Core version</strong>: {frontMatter.meta.min_core_version} and newer</li>
+    <li><strong>dbt Cloud support</strong>: {frontMatter.meta.cloud_support}</li>
+    <li><strong>Minimum data platform version</strong>: {frontMatter.meta.min_supported_version}</li>
+    </ul>
+
+
+<h2> Installing {frontMatter.meta.pypi_package} </h2>
+
+pip is the easiest way to install the adapter:
+
+<code>pip install {frontMatter.meta.pypi_package}</code>
+
+<p>Installing <code>{frontMatter.meta.pypi_package}</code> will also install <code>dbt-core</code> and any other dependencies.</p>
+
+<h2> Configuring {frontMatter.meta.pypi_package} </h2>
+
+<p>For further info, refer to the GitHub repository: <a href={`https://github.com/${frontMatter.meta.github_repo}`}>{frontMatter.meta.github_repo}</a></p>
+
+
+## Connecting to Infer with **dbt-infer**
+
+Infer allows you to perform advanced ML Analytics within SQL as if native to your data warehouse.
+To do this Infer uses a variant called <a href="https://docs.getinfer.io/docs/reference">SQL-inf</a> 
+which defines as set of primitive ML commands from which 
+you can build advanced analysis for any business use case.
+Learn more about SQL-inf and Infer <a href="https://docs.getinfer.io/">here</a>.
+
+The `dbt-infer` package allow you to use SQL-inf easily within your DBT models. 
+You can read more about the `dbt-infer` package <a href="https://dbt.getinfer.io/">here</a>.
+
+Before using SQL-inf in your DBT models you need to setup an Infer account and generate an API-key for the connection.
+You can read how to do that <a href="https://dbt.getinfer.io/docs/getting_started#sign-up-to-infer">here</a>.
+
+The profile configuration in `profiles.yml` for `dbt-infer` should look something like this:
+
+<File name='~/.dbt/profiles.yml'>
+
+```yaml
+<profile-name>:
+  target: <target-name>
+  outputs:
+    <target-name>:
+      type: infer
+      url: "<infer-api-endpoint>"
+      username: "<infer-api-username>"
+      apikey: "<infer-apikey>"
+      data_config:
+        [configuration for your underlying data warehouse]  
+```
+
+</File>
+
+Note that you need to also have installed the adapter package for your underlying data warehouse.
+For example, if your data warehouse is BigQuery then you need to also have installed the appropriate `dbt-bigquery` package.
```

Comment:
can you tell me more about how this works? dbt users would love to have a multi-adapter project, but they are still a ways away. 

based on your instructions, my guess is that first they must provide Infer with access to their underlying data warehouse. Otherwise, users would also have to add their DW creds in their `profiles.yml`, right?

unless it is via `data_config`? interesting...

Reply:
Spoke about it earlier but adding here for completeness :-)
That is indeed what `data_config` is for - that is where you provide your typical data warehouse `profiles` configuration. That is then used to instantiate the underlying DWH adapter, which the infer adapter will use to execute dbt models and fetch data.
I'll add a bit more clarification of that point in the docs.